doc/modules/mixture.rst

```diff
@@ -266,9 +292,9 @@ Pros
    stability and less tuning.
 
 :No need to specify the number of components: only an upper bound of
-   this number needs to be provided. Note however that the DPMM is not
-   a formal model selection procedure, and thus provides no guarantee
-   on the result.
+   this number needs to be provided. Note however that the Dirichlet process
+   Gaussian mixture is not a formal model selection procedure, and thus
+   provides no guarantee on the result.
```

Comment:
One could argue that this is actually a formal model selection procedure but that does not mean that it is that useful in practice ;) I would rephrase this last sentence to the following:

Note however that the "ideal" number of active components is very application specific and is typically ill-defined in a data exploration setting. The effective number of active components often depends a lot on the value for the weights concentration prior.